bob/src/api/http.rs

```diff
@@ -568,6 +584,16 @@ async fn alien(_bob: &State<BobServer>) -> &'static str {
     "alien"
 }
 
+#[post("/alien/sync")]
+async fn sync_alien_data(bob: &State<BobServer>) -> Result<StatusExt, StatusExt> {
+    let backend = bob.grinder().backend().inner();
+    let (_, adc) = backend
+        .disk_controllers()
+        .ok_or_else(not_acceptable_backend)?;
+    adc.detach_all().await?;
```

Comment:
Is this a correct command? It looks like it is detaching (not syncing) all blobs in Bob (not just aliens)

Reply:
`adc` is `AliensDiskController`, so we are detaching (making inactive) all alien blobs

Reply:
Can you make variable name more intuitive? E.g. `aliensDiskController`

Reply:
Done, changed all occurrences of `adc` in `http.rs` to `alien_disk_controller`